pihole-google.txt

```diff
@@ -8198,4 +8198,23 @@ www.tofu.jibemobile.com
 www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
-yellowstone.jibemobile.com
\ No newline at end of file
+yellowstone.jibemobile.com
+# More
+google.co1.qualtrics.com
+googlefibersurveys.qualtrics.com
+mobilecrashreporting.googleapis.com
+3.android.pool.ntp.org
+0.android.pool.ntp.org
+1.android.pool.ntp.org
+2.android.pool.ntp.org
+4.android.pool.ntp.org
+5.android.pool.ntp.org
+6.android.pool.ntp.org
+7.android.pool.ntp.org
+8.android.pool.ntp.org
+android.pool.ntp.org
+9.android.pool.ntp.org
+media.fastclick.net
+mt34.mtree.com
+wurfl.io
+blueadvertise.com
```

Comment:
Great find!

```suggestion
mobilecrashreporting.googleapis.com
```

The others are not Google domains:

https://www.whois.com/whois/qualtrics.com
https://www.whois.com/whois/ntp.org
https://www.whois.com/whois/fastclick.net
https://www.whois.com/whois/mtree.com
https://www.whois.com/whois/wurfl.io
https://www.whois.com/whois/blueadvertise.com